[PATCH] [MGMTD] This error happens when we try to write to a socket or pipe

From: sudhanshukumar22 <sudhanshu.kumar@broadcom.com>
Date: Wed, 5 Mar 2025 00:55:58 -0800
Subject: [PATCH] [MGMTD] This error happens when we try to write to a socket or pipe
 which is already closed by peer.  For e.g., the other side may have been
 killed during warmboot or using pkill. Hence, changing the log level to INFO
---
 lib/mgmt_msg.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/mgmt_msg.c b/lib/mgmt_msg.c
index 8f22b3123..4e6964532 100644
--- a/lib/mgmt_msg.c
+++ b/lib/mgmt_msg.c
@@ -28,6 +28,8 @@
 #define MGMT_MSG_ERR(ms, fmt, ...)                                             \
 	zlog_err("%s: %s: " fmt, (ms)->idtag, __func__, ##__VA_ARGS__)
 
+#define MGMT_MSG_INFO(ms, fmt, ...)                                             \
+	zlog_info("%s: %s: " fmt, (ms)->idtag, __func__, ##__VA_ARGS__)
 DEFINE_MTYPE(LIB, MSG_CONN, "msg connection state");
 
 /**
@@ -70,7 +72,7 @@ enum mgmt_msg_rsched mgmt_msg_read(struct mgmt_msg_state *ms, int fd,
 		}
 		if (n <= 0) {
 			if (n == 0)
-				MGMT_MSG_ERR(ms, "got EOF/disconnect");
+				MGMT_MSG_INFO(ms, "got EOF/disconnect");
 			else
 				MGMT_MSG_ERR(ms,
 					     "got error while reading: '%s'",
